### PR TITLE
Merge notifier config into sovereign config

### PR DESCRIPTION
**Pull request by axenteoctavian**
## Reasoning behind the pull request
- [TASK](https://multiversxlabs.atlassian.net/browse/MX-14918)
- 
- 
  
## Proposed changes
- merge notifierConfig.toml into sovereignConfig.toml and refactor
- 
- 

## Testing procedure
- running sovereign node
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: cmd/sovereignnode/systemTestDemo/README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ INFO [2023-04-24 17:08:42.024] [..overeign-process] [0/0/106/(END_ROUND)] found
 ```
 
 Inside this folder, there is a `main.go` script which sends a transaction to one of the subscribed sovereign addresses.
-The address is specified by `SubscribedAddresses` field from this config
-file `mx-chain-go/cmd/sovereignnode/config/notifierConfig.toml`
+The address is specified by `Addresses` field from `SubscribedEvents` struct from this config
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/README.md:35]:
```suggestion
The address is specified by `Addresses` field from `SubscribedEvents` struct for identifier = "deposit" from this config
```

File: cmd/sovereignnode/sovereignNodeRunner.go [hunk 2]
```diff
@@ -1785,16 +1785,16 @@ func createWhiteListerVerifiedTxs(generalConfig *config.Config) (process.WhiteLi
 }
 
 func createIncomingHeaderProcessor(
-	config *sovereignConfig.NotifierConfig,
+	config *sovereignConfig.SovereignConfig,
```
[the remark this answers is not in the document]
Comment (axenteoctavian) [reply cmd/sovereignnode/sovereignNodeRunner.go]: `*config.NotifierConfig`

File: cmd/sovereignnode/sovereignNodeRunner.go [hunk 3]
```diff
@@ -1812,13 +1812,13 @@ func createIncomingHeaderProcessor(
 }
 
 func createSovereignWsReceiver(
-	config *sovereignConfig.NotifierConfig,
+	config *sovereignConfig.SovereignConfig,
```
[the remark this answers is not in the document]
Comment (axenteoctavian) [reply cmd/sovereignnode/sovereignNodeRunner.go]: `*config.NotifierConfig`

**Review by sasurobert — DISMISSED** [review]

File: cmd/sovereignnode/systemTestDemo/README.md [hunk 4]
```diff
@@ -95,7 +95,7 @@ triggers the transmission of blocks, starting from an arbitrary nonce, with inco
 Each event includes the transfer of an NFT and an ESDT token. The periodic transmission consists of 2 NFTs
 (ASH-a642d1-01 & ASH-a642d1-02) and one ESDT token (WEGLD-bd4d79).
 
-To verify the success of token transfers, you can utilize the sovereign proxy for the subscribed address, which can be
+To verify the success of token transfers, you can utilize the sovereign proxy for the subscribed address where identifier = "deposit", which can be
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/README.md:98]: This file needs to be reformatted. Line passed 120 char limits. After reformat, all lines need to be re-arranged

**Review by mariusmihaic — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**